pyproject.toml

```diff
@@ -16,7 +16,7 @@ readme = "README.md"
 requires-python = ">=3.12"
 license = "Apache-2.0"
 dependencies = [
-    "llama-stack",
+    "llama-stack>=0.2.5",
```

Comment:
I'm not sure if the module approach is compatible with LLS below 0.2.17.
If not, should we pin this to >=0.2.17?